src/components/auto-scale-cannon-physics-body.js

```diff
@@ -0,0 +1,39 @@
+function almostEquals(u, v, epsilon) {
+  return Math.abs(u.x - v.x) < epsilon && Math.abs(u.y - v.y) < epsilon && Math.abs(u.z - v.z) < epsilon;
+}
+
+function debounce(fn, delay) {
```

Comment:
going to rewrite it to avoid `debounce` because (I think) having the `debounced` function call `requestAnimationFrame` or `setTimeout` and `clearTimeout` rapidly is undesirable.